app.js

```diff
@@ -299,6 +302,37 @@ function go() {
 
   let sites = _.filter(data.sites, validSiteFilter);
 
+  sites = sites.map(site => {
+    site.backends.sort((b1, b2) => {
+      const p1 = pathOf(b1);
+      const p2 = pathOf(b2);
+      if (p1 < p2) {
+        return -1;
+      } else if (p2 > p1) {
+        return 1;
+      } else {
+        return 0;
+      }
+    });
+    site.backendGroups = [];
+    let lastPath = null;
```

Comment:
I don't see this reassigned. So it could be const, but also the conditional below will always be true (unless the first part is null).